src/main/kotlin/org/digma/intellij/plugin/ui/list/insights/SpanPanelsUtil.kt

```diff
@@ -159,7 +159,7 @@ fun buildButtonToJaeger(
     if (jaegerBaseUrl.isNullOrBlank() || traceSamples.isNullOrEmpty()) {
         return null
     }
-    val filtered = traceSamples.filter { x -> x!!.hasTraceId() }
+    val filtered = traceSamples.filter { x -> x!=null &&  x!!.hasTraceId() }
```

Comment:
this should work fine:
`val filtered = traceSamples.filter { x -> x != null &&  x.hasTraceId() }`
you should remove `!!` before `.hasTraceId()` as at this point we are already sure that `x` is not null

Reply:
I've just committed this fix